src/Api/Projects.php

```diff
@@ -1035,13 +1035,17 @@ public function languages($project_id)
 
     /**
      * @param int|string $project_id
-     * @param array      $parameters
+     * @param array      $parameters {
+     *
+     *     @var string $search         Return list of forks matching the search criteria (optional)
```

Comment:
If we are going to document this, we need to document all the available parameters.

Reply:
I can try that. I wanted to get the minimum PR that would work and then have it reviewed by maintainers (aka you) to be sure that this was the right approach. I'll try to add and document the other options.

Reply:
I've done it now based on the current code (I took the `all` method as a reference) and the information I found here: https://docs.gitlab.com/ee/api/projects.html#list-forks-of-a-project